modules/serverpod_auth/serverpod_auth_server/lib/src/business/config.dart

```diff
@@ -47,6 +47,15 @@ typedef PasswordHashValidator = Future<bool> Function(
   void Function(Object e)? onError,
 });
 
+/// Enum to define the sign-out options for a user.
```

Comment:
Suggestion: Change to "Enum to define the sign-out behavior for the legacy sign out endpoint".